<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="inputGroup">
      <property name="title">
       <string>Introducere produs</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nume produs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Cantitate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="quantitySpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Pret</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="priceDoubleSpinBox">
         <property name="decimals">
          <number>2</number>
         </property>
         <property name="maximum">
          <double>100000000.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>Adauga produs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editButton">
         <property name="text">
          <string>Modifica produs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>Sterge produs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="filterGroup">
         <property name="title">
          <string>Filtreaza produsele</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLineEdit" name="filterLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="filterButton">
            <property name="text">
             <string>Aplica filtrul</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="resetFilterButton">
            <property name="text">
             <string>Reseteaza filtrul</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTableView" name="productTableView">
      <property name="minimumSize">
       <size>
        <width>541</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSTOC">
    <property name="title">
     <string>STOC</string>
    </property>
   </widget>
   <addaction name="menuSTOC"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
